demo/src/static/workers/flight-path-data-decoder.js

```diff
@@ -0,0 +1,55 @@
+importScripts('./util.js');
+const FLUSH_LIMIT = 20000;
+let result = [];
+let index = 0;
+let count = 0;
+
+onmessage = function(e) {
```

Comment:
`e` is not a good parameter name. Simply spelling this out will make this code easier to read.